easybuild/easyblocks/s/suitesparse.py

```diff
@@ -99,7 +99,7 @@ def configure_step(self):
                     # for variables in cfgvars, substiture lines assignment 
                     # in the file, whatever they are, by assignments to the
                     # values in cfgvars
-                    line = re.sub(r"^\s*(%s\s*=\s*).*$" % var,
+                    line = re.sub(r"^\s*(%s\s*=\s*).*\n$" % var,
                                   r"\1 %s # patched by EasyBuild" % val,
```

Comment:
the newline needs to be restored, see https://github.com/ocaisa/easybuild-easyblocks/pull/6